### PR TITLE
docs: ncip-20 draft

### PR DESCRIPTION
**Pull request by jaeho0103**
https://github.com/planetarium/NCIPs/pull/18

### CODE REVIEW

File: NCIP/ncip-20.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        
+        
+        | Distribution | Timing | Source of Distributed NCG |
+        | --- | --- | --- |
+        | 1st | Mid-August | NCG used for Adventure Boss from July to mid-August |
+        | 2nd | Mid-September | NCG used for adventure bosses from mid-August to mid-September + 9C gameplay NCG from July to mid-September |
+        | 3rd | Mid-October | NCG used for adventure bosses from mid-September to mid-October + 9C gameplay NCG from September to mid-October |
+        - 9C gameplay NCG spent on the following:
+            - Shop Fee
+            - Item Enhancement
```
[the remark this answers is not in the document]
Comment (jaeho0103) [reply NCIP/ncip-20.md:43]: 넵 수정하겠습니다.

File: NCIP/ncip-19.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+Players can transfer their Staking assets to another Guild at any time. This is called Redelegation, and can be transferred to another Guild without a lockup period, minimizing losses when transferring to another Guild.
+
+## Earn NCG
+
+Guilds have the right to acquire NCG spent by players belonging to the Guild. This includes:
+
+- Buy
+- Sell
+- Craft
+- Enhancement
```
[the remark this answers is not in the document]
Comment (jaeho0103) [reply NCIP/ncip-19.md:63]: 넵 수정하겠습니다.

File: NCIP/ncip-19.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
+---
+NCIP: 19
+Title: Guild System
+Status: Draft
+Type: Core
+Author: Suho Lee <suho@planetariumhq.com.>.
```
Comment (Namyujeong) [inline NCIP/ncip-19.md:6]: auther가 19, 20 모두 수호로 돼있어서 이것도 재호로 바꿔주셔야

**Review by Namyujeong — APPROVED** [review]

**Comment by jaeho0103** [comment]
As there were no negative opinions or objections regarding the proposal, we will proceed with the first round of manual distribution. We appreciate your continued interest and feedback.

**Pull request merged**